syft/generic/object_storage.py

```diff
@@ -70,15 +70,21 @@ def get_obj(self, obj_id: Union[str, int]) -> object:
         Returns:
             Object with id equals to `obj_id`.
         """
-
+        obj = None
         try:
-            obj = self._objects[obj_id]
-        except KeyError as e:
-            if obj_id not in self._objects:
-                raise ObjectNotFoundError(obj_id, self)
-            else:
-                raise e
-
+            # if object is a tesor
+            obj = self._objects.get(obj_id, None)
```

Comment:
You don't want to use self.object_store.get_obj because it raises an exception?

Reply:
yep... currently I get 
E   RecursionError: maximum recursion depth exceeded

Reply:
What's the stack trace for that error? I probably messed something up when I extract `ObjectStore`.

Reply:
@karlhigley  I was reproduce the stack trace now that tests are working again,
here you go!
![image](https://user-images.githubusercontent.com/13193437/83071853-c3665680-a03b-11ea-9464-3a05196abe1e.png)

Reply:
Based on the line numbers, I think the issue must be coming from [around here](https://github.com/OpenMined/PySyft/pull/3538/files#diff-f1b6d98d314c69d43920489c6a7e469cL74-R75), but I don't see code that looks like the snippets in the stack trace. Not 100% sure how it ends up calling `get_obj()` again inside `get_obj()`. Might be worth experimenting with the `try/except` version that uses `self._objects[obj_id]` instead of `self._objects.get(obj_id, None)` to see if that helps.